tests/foreman/api/test_capsulecontent.py

```diff
@@ -1894,3 +1894,63 @@ def test_positive_reclaim_space(
         assert (
             reclaim_doc['apis'][0]['api_url'] == '/katello/api/capsules/:id/content/reclaim_space'
         ), 'Documented path did not meet the expectation.'
+
+
+class TestPodman:
+    """Tests specific to using podman push/pull on Satellite
+
+    :CaseComponent: Repositories
+
+    :team: Phoenix-content
+    """
+
+    @pytest.fixture(scope='class')
+    def enable_podman_capsule(module_product, module_capsule_configured):
+        """Enable base_os and appstream repos on the sat through cdn registration and install podman."""
+        module_capsule_configured.register_to_cdn()
+        if module_capsule_configured.os_version.major > 7:
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel_bos']['id'])
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel_aps']['id'])
+        else:
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhscl']['id'])
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel']['id'])
+        result = module_capsule_configured.execute(
+            'dnf install -y --disableplugin=foreman-protector podman'
+        )
+        assert result.status == 0
+
+    def test_negative_podman_capsule_push(
+        self,
+        module_target_sat,
+        module_product,
+        module_org,
+        module_lce,
+        enable_podman_capsule,
+        module_capsule_configured,
+    ):
+        """Attempt to push a Podman container to a Capsule/Smart Proxy
```

Comment:
I don't want to be too sctrict about using correct terminology, however it would be useful for further maintenance. We should not mix up terms _container_ and _image_.  The shortest but not entirely accurate explanation is that image is kind of template and container is an instance of isolated environment made from the template.
There are also multiple container engines, not just podman (docker, containerd...).
In this test we are working only with _images_. To reflect different container engines we can use collocation "_container images_" or maybe "_OCI images_"